### PR TITLE
Fix release note button clicking crash

### PR DESCRIPTION
**Pull request by aviraxp**
As for context, we should set this flag. Also check for null when there is no internet.

### CODE REVIEW

File: app/src/main/java/com/topjohnwu/magisk/InstallFragment.java [hunk 1]
```diff
@@ -100,7 +100,11 @@ public void onDownloadDone(Uri uri) {
                         magiskManager.magiskLink,
                         Utils.getLegalFilename(filename)))
                 .setNeutralButton(R.string.release_notes, (dialog, which) -> {
-                    magiskManager.startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(magiskManager.releaseNoteLink)));
+                    if (magiskManager.releaseNoteLink != null) {
+                        Intent openReleasNoteLink = new Intent(Intent.ACTION_VIEW, Uri.parse(magiskManager.releaseNoteLink));
```
Comment (TheComputerGuy96) [inline app/src/main/java/com/topjohnwu/magisk/InstallFragment.java]: Typo

File: app/src/main/java/com/topjohnwu/magisk/InstallFragment.java [hunk 2]
```diff
@@ -100,7 +100,11 @@ public void onDownloadDone(Uri uri) {
...
+                        openReleasNoteLink.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                        magiskManager.startActivity(openReleasNoteLink);
```
Comment (TheComputerGuy96) [inline app/src/main/java/com/topjohnwu/magisk/InstallFragment.java]: Typo

File: app/src/main/java/com/topjohnwu/magisk/InstallFragment.java [hunk 3]
```diff
@@ -100,7 +100,11 @@ public void onDownloadDone(Uri uri) {
                         magiskManager.magiskLink,
                         Utils.getLegalFilename(filename)))
                 .setNeutralButton(R.string.release_notes, (dialog, which) -> {
-                    magiskManager.startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(magiskManager.releaseNoteLink)));
+                    if (magiskManager.releaseNoteLink != null) {
+                        Intent openReleasNoteLink = new Intent(Intent.ACTION_VIEW, Uri.parse(magiskManager.releaseNoteLink));
+                        openReleasNoteLink.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```
Comment (TheComputerGuy96) [inline app/src/main/java/com/topjohnwu/magisk/InstallFragment.java]: Typo

**Review by TheComputerGuy96 — CHANGES_REQUESTED** [review]
You made some typos

**Comment by topjohnwu** [comment]
Thanks, this crash happens on Samsung only too :)

**Comment by aviraxp** [comment]
No, it happens on almost every device I have.

**Pull request merged**